app/models/private_childcare_provider.rb

```diff
@@ -27,6 +27,10 @@ def urn
     provider_urn
```

Comment:
i'd be tempted to store this under a new different field eg `ey_urn` or `childcare_urn` if they are actually a different from `urn`